.github/workflows/openssf_scorecard.yaml

```
@@ -0,0 +1,44 @@
+name: OpenSSF Scorecard
+
+permissions: {}
+
+on:
+  schedule:
+    - cron: '30 1 * * 6'
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  analysis:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          persist-credentials: false
+      - name: Run analysis
+        uses: ossf/scorecard-action@0864cf19026789058feabb7e87baa5f140aac736 # v2.3.1
+        with:
+          results_file: results.sarif
+          results_format: sarif
+          repo_token: ${{ secrets.SCORECARD_READ_TOKEN }}
```

```suggestion
          # repo_token: ${{ secrets.SCORECARD_READ_TOKEN }}
```
`repo_token` is only needed if Branch-Protection check needs to be enabled. We can do without this since maintaining this token might have operational issues.